test/verify/check-storage-stratis

```diff
@@ -438,36 +438,43 @@ class TestStorageStratis(StorageCase):
 
 
 @skipImage("No Stratis", "debian-*", "ubuntu-*", "arch")
-@skipImage("No Stratis on demand", "rhel-8-*")
 class TestStoragePackagesStratis(PackageCase, StorageCase):
 
     def testStratisOndemandInstallation(self):
         m = self.machine
         b = self.browser
 
-        m.execute("systemctl stop stratisd")
-        self.write_file("/etc/cockpit/storaged.override.json",
-                        """{ "config": { "stratis_package": "fake-stratisd" } }""")
-        self.createPackage("fake-stratisd", "999", "1")
-        self.enableRepo()
-        m.execute("pkcon refresh")
+        # RHEL 8 should not offer installation of Stratis from Cockpit
+        # itself.
+        #
+        ondemand_stratis = "rhel-8" not in m.image
+
+        m.execute("systemctl stop stratisd && dnf remove -y stratisd stratis")
```

Comment:
Please avoid `&&`, it's semantics are really confusing. `m.execute()` runs with `set -e`, so using `;` does the right thing.